CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2023-04-04
+### Added
+- `RoomHandler.onAccessOfferChange` is now available to get notified of activated Site Offers (#75).
+- `PlatformClient.inquireForGPSActivatedOffer` is now available to inquire for available Site Offers when outside of a call (#75).
+- `Position` can be now compared using the '==' operator to know if both positions have same longitude, latitude, heading and timestamp (#75).
+- `Position.toString` will now show longitude, latitude, heading and timestamp (#75).
+- `Position.timeSinceInMs` is now an available convenience to know the amount of time elapsed since `Position.timestamp` and now (#75).
+- `Position.distanceInMetersWith` is now an available convenience to know the between two Positions (#75).
```

Comment:
I'm not a huge fan of the names, but no biggie.

A few comments:
*  If `onAccessOfferChange` is an optional event (similar to `onConnectionLost`), should it go on the `Room` instead of the `RoomHandler`? Does the event only apply to location-based offers or will it trigger when the Agent activates a promotion? `onAccessOfferApplied` sounds better to me.
* Instead of `inquireForGPSActivatedOffer`, why not `updateLocation` (similar to the method on the `Room`)? 
* "distance between" or "distance to/from" is more common than "distance with". And why meters? I think our app uses the Imperial system right now, correct (although we eventually want to make that a user preference)?

Reply:
about the `onAccessOfferChage`: I did it exactly how you propose it to be done. It is in `Room` but the diff is a little miss-leading because it hides some of the code.

Reply:
I can change the `inquireForGPSActivatedOffer` to `updateLocation`... it just sounds inaccurate to me because it doesn't update anything on the backend side.

Reply:
about `distanceWith`... yep! `distanceFrom` sounds a lot better.

Reply:
Meters because: 1. the plugins works with meters 2. the function example I've seen was in km and I got a little lazy with that I guess 3. Legacy worked with meters 4. the whole world works with meters... including Nasa 5. metric system is so much easier: it is less error prone 6. the imperial system... I mean! I thought about using it and the idea didn't fit well with my whole being 😄

Reply:
>  it just sounds inaccurate to me because it doesn't update anything on the backend side 
Are you sure? It looks like `SiteAccessManager.processGPSState` does quite a bit, including updating the `currentLocation` user property.

Reply:
> Legacy worked with meters

I didn't know that. When I browse location offers in the Aira - Legacy and Aira Explorer iOS apps, distances are shown in miles. Are you going to change that? If so, I'd advise against that. Most of our Explorers are in the US and the US uses miles.

In the future, allowing Explorers to choose their units makes a ton of sense, but I don't think that's a priority at the moment.